es-bs-base/scss/vue/components/_slider.scss

```diff
@@ -0,0 +1,49 @@
+@use "../variables";
+@use "../../variables" as v;
+@use "../../functions";
+@use 'sass:color';
+
+/* stylelint-disable scss/dollar-variable-pattern */
+@import '~vue-slider-component/lib/theme/default';
+@import '~vue-slider-component/lib/styles/dot';
+@import '~vue-slider-component/lib/styles/mark';
+@import '~vue-slider-component/lib/styles/slider';
+
+.es-slider {
+  padding: 0 !important;
+
+  .vue-slider-process {
+    background: variables.$b-slider-background;
+  }
+
+  .vue-slider-rail {
+    height: variables.$b-slider-rail-height;
+  }
+
+  .vue-slider-dot-handle {
```

Comment:
It looks like the Figma design specifies a different shadow than the dot handle is getting right now - maybe we can put that one here.